src/drive/web/modules/drive/RenameInput.jsx

```diff
@@ -18,6 +18,19 @@ const updateFileNameQuery = async (client, file, newName) => {
   })
 }
 
+const isSanitized = fileName => {
+  if (fileName === '.' || fileName === '..') {
+    return false;
+  }
+  let result = true;
+  '/\x00\n\r'.split('').forEach(forbiddenCharacter => {
```

Comment:
More forbidden characters exist. You can find a (_should be complete_) list here : https://github.com/cozy-labs/cozy-desktop/blob/master/core/incompatibilities/platform.js#L110-L160

Reply:
@Ldoppea This list is about Desktop. Not Cozy "in general". We've specific forbidden characters for different OS, but we do not add all these forbidden characters to Cozy (yet?). This is a discussion we had multiple times, but we can launch it again to see if something has changed. 

Cozy-Stack will throw an error only for this small list.

Reply:
What would be the benefits of maintaining a different list on OS and on Cozy? Wouldn't it be easier to have the more restrictive list everywhere?

> This is a discussion we had multiple times

Do we have a SEC somewhere? If not maybe we should write one next time we will discuss this. This is the kind of question that would come over and over every time new devs will encounter those codes 😅

Reply:
We want people to be able to synchronize all of their files from their computer to Cozy. For example, we have seen macOS users to have office document with a ? in the filename. And cozy-desktop should be able to upload them to the Cozy in background, with no user interaction.

Reply:
this thread is outdated, I resolve this conversation